packages/kit/src/exports/node/index.js

```diff
@@ -59,7 +58,13 @@ function get_raw_body(req, body_size_limit) {
 
 				size += chunk.length;
 				if (size > length) {
-					controller.error(new Error('content-length exceeded'));
+					req.destroy(
+						new Error(
+							`request body size exceeded ${
+								content_length ? "specified 'content-length'" : 'allowed by the server'
```

Comment:
this would read "request body size exceeded allowed by server" which isn't a sentence that makes sense. we could change it like this:
```suggestion
								content_length ? "'content-length'" : 'BODY_SIZE_LIMIT'
```

Reply:
@benmccann thank you